mapadroid/worker/WorkerQuests.py

```diff
@@ -635,12 +638,14 @@ def _current_position_has_spinnable_stop(self, timestamp: float):
                 if fort_type == 0:
                     self._db_wrapper.delete_stop(latitude, longitude)
                     self.logger.warning("Tried to open a stop but found a gym instead!")
+                    self._spinnable_data_failcount = 0
```

Comment:
This might be the culprit.

We are looping over all pokestops and gyms we found in the GMO, but if we find a gym close by, we try and delete the pokestop at that location, and claim we tried to open a gym.

Actually I'm curious why this check is even here. This is the "are we at a place where there's a stop in the gmo" - not "we have tried to click a stop" ... anyhow. if gmo finds a gym here, it will fail spinnable check and reset spinnable

Reply:
But this is after the coords check and max accepted diff there is 0.00003. That gym would have been almost on top of pokestop, like ~4 meters MAX apart.

Reply:
I haven't followed this. But in my locality we have two gyms where people have moved pokestops right on top. Literally they show as underneath.  So these situations do exist.  These stops fail quest scan, but if you msg me on discord I will send you long,lat so you can see

Reply:
That log line is misleading. We're not yet trying to open anything there. I want to reset the counter every time we get a "defined" result. So finding a gym where we expected a stop is a defined result. The stop being disabled/closed/on cooldown is a defined result.

On the other hand - not finding the stop at all or having "no map data" is not a defined result, because it could possibly be some type of hiccup.